Assembler.Base/MiddleFrameHandler.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using Assembler.Core;
+using Assembler.Core.Entities;
+
+namespace Assembler.Base
+{
+    public class MiddleFrameHandler<TFrame, TMessage> : BaseFrameHandler<TFrame, TMessage>
```

Comment:
Please see the comments if the InitialFrameHandler and apply them here too

Reply:
f